screens/stores/StoreListScreen.js

```diff
@@ -18,12 +24,33 @@ class StoreListScreen extends React.Component {
     });
   };
 
+  updateSearch = searchStr => {
+    this.setState({
+      searchStr,
+      filteredStores: this.state.allStores.filter(this.filterStore(searchStr))
+    });
+  };
+
+  filterStore(searchStr) {
+    return store => {
+      return store.name.toLowerCase().includes(searchStr.toLowerCase());
+    };
+  }
+
   render() {
+    const { searchStr } = this.state;
+
     return (
       <StoreModal>
+        {/* Search bar */}
+        <SearchBar
+          placeholder="Type Here..."
```

Comment:
Maybe replace placeholder with something more intuitive like "enter store name here" or "Store name"? But if you feel like everything else looks intuitive then feel free to keep :0

Reply:
+1 Change placeholder text to "Search by Store Name" to reflect the Figma and make it explicit what exactly users can search by. It seems like a nit but this is very important for real usability!